csse_covid_19_data/README.md

```diff
@@ -339,6 +339,7 @@ Date: Location | Change | Files affected | Reason/Other notes | Source
 * May 09, 2022 | Ohio, US | To match Ohio’s new reporting standards, the Ohio death time series is being overwritten with date of death data, and the entire time series will be updated with each time Ohio updates their death data. This is expected to occur approximately once per week. | [Data Source](https://coronavirus.ohio.gov/static/dashboards/COVIDDeathData_CountyOfResidence.csv)
 * May 09, 2022 | Tennessee, US | Corrected from 2020-03-12 to 2022-04-30 | [Data Source](https://www.tn.gov/content/dam/tn/health/documents/cedep/novel-coronavirus/datasets/Public-Dataset-County-New.XLSX)
 * May 09, 2022 | North Dakota, US | Corrected from 2020-03-09 to 2022-05-05 | [Data Source](https://www.health.nd.gov/sites/www/files/documents/Files/MSS/coronavirus/charts-data/PublicUseData.csv)
+* May 16, 2022 | Austria | Corrected from 2020-01-03 to 2022-05-11 | [Data Source](https://info.gesundheitsministerium.gv.at/data/timeline-faelle-bundeslaender.csv)
```

Comment:
At least for deaths, the data source seems to be https://covid19-dashboard.ages.at/?l=en (which is also an official source and actually the correct one), not the cited https://info.gesundheitsministerium.gv.at/data/timeline-faelle-bundeslaender.csv (which is a semi-official collection of per-state data and is still missing some late registrations that the AGES has).